docs/guide.md

```diff
@@ -352,6 +352,9 @@ When you are not allowed or do not want to run packit command from the upstream
 git repository, you can run commands from the cloned dist-git repository in the
 same way as you do from upstream.  Make sure, that you specify the
 `upstream_project_url` in your configuration.
+You don't need to specify the upstream project tarball URL but just the upstream
+ project repo URL in the Packit configuration file because the tarball URL is taken 
+ from the specfile when needed.
```

Comment:
I don't think this should be here. This paragraph seems to talk about SRPM building.